app/src/main/java/com/mapbox/maps/testapp/examples/MapViewCustomizationActivity.kt

```diff
@@ -0,0 +1,84 @@
+package com.mapbox.maps.testapp.examples
+
+import android.os.Bundle
+import android.widget.LinearLayout
+import androidx.appcompat.app.AppCompatActivity
+import com.mapbox.geojson.Point
+import com.mapbox.maps.*
+import com.mapbox.maps.testapp.R
+import kotlinx.android.synthetic.main.activity_map_view_customization.*
+
+/**
+ * Example of customizing your [MapView].
+ * This example will show how to create [MapView] both programmatically and from xml
+ * and apply various options.
+ */
+class MapViewCustomizationActivity : AppCompatActivity() {
+
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    setContentView(R.layout.activity_map_view_customization)
+    configureMapViewFromXml()
+    configureMapViewFromCode()
+  }
+
+  private fun configureMapViewFromCode() {
+    val mapboxMapOptions = MapboxMapOptions(this)
```

Comment:
nit: I would prefer to use apply block here.

Reply:
👍